### PR TITLE
fix: Add command line ADM target support:

### PR DESCRIPTION
**Pull request by pleacu**
java -Datlasmap.adm.path=/home/pleacu/Downloads/rat1.adm -jar atlasmap-standalone.jar

Fixes: #755

### CODE REVIEW

File: lib/core/src/main/java/io/atlasmap/core/AtlasUtil.java [hunk 1]
```diff
@@ -461,4 +464,34 @@ else if (element.isDirectory())
         }
         return;
     }
+
+    public static void copyFile(Path sourcePath, Path destPath) throws IOException {
```
Comment (apupier) [inline lib/core/src/main/java/io/atlasmap/core/AtlasUtil.java]: what about using Files.copy(Path, Path, CopyOption...)?
https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#copy-java.nio.file.Path-java.nio.file.Path-java.nio.file.CopyOption...-
I think it is doing the same thing, isn't it?

File: lib/service/src/main/java/io/atlasmap/service/AtlasService.java [hunk 2]
```diff
@@ -113,6 +119,42 @@ public void onUpdate(AtlasLibraryLoader loader) {
         });
     }
 
+    /**
+     * The user has specified an ADM file on inception.  Clear current mappings and process
```
Comment (apupier) [inline lib/service/src/main/java/io/atlasmap/service/AtlasService.java]: exra space between "inception." and "Clear"

File: lib/service/src/main/java/io/atlasmap/service/AtlasService.java [hunk 3]
```diff
@@ -113,6 +119,42 @@ public void onUpdate(AtlasLibraryLoader loader) {
...
+                Paths.get(mappingFolderPath.toAbsolutePath().toString() + File.separator + atlasmapCatalogName));
+
+            extractCompressedCatalog();
+
+        } catch (Exception e) {
+            LOG.error("Error processing the AtlasMap catalog file " + atlasmapAdmPath + "\n" +
+                e.getMessage(), e);
+            throw new WebApplicationException(e.getMessage(), e, Status.INTERNAL_SERVER_ERROR);
+        }
+        System.setProperty(ATLASMAP_ADM_PATH, "");
```
Comment (apupier) [inline lib/service/src/main/java/io/atlasmap/service/AtlasService.java]: why are you setting the value to empty value? it means that after launch and first use the property provided by the user is overridden? Why not keeping it to know on which transformation it was initially launched?

File: lib/service/src/main/java/io/atlasmap/service/AtlasService.java [hunk 4]
```diff
@@ -214,13 +260,7 @@ public Response resetMappings() {
...
 
         try {
-            for (File mappingFile : mappings) {
-                if (mappingFile.exists()) {
-                    if (!mappingFile.delete()) {
-                        LOG.warn("Unable to delete mapping file " + mappingFile.toString());
-                    }
-                }
-            }
+            AtlasUtil.deleteDirectoryContents(mappingFolderPath.toFile());
```
Comment (apupier) [inline lib/service/src/main/java/io/atlasmap/service/AtlasService.java]: what about using Files.deleteIfExists(Path) or Files.delete(Path)?

File: lib/core/src/main/java/io/atlasmap/core/AtlasUtil.java (hunk 1 above)
Comment (pleacu) [reply lib/core/src/main/java/io/atlasmap/core/AtlasUtil.java]: I was having trouble with StandardCopyOption.REPLACE_EXISTING - I can try it again.

File: lib/service/src/main/java/io/atlasmap/service/AtlasService.java (hunk 2 above)
Comment (pleacu) [reply lib/service/src/main/java/io/atlasmap/service/AtlasService.java]: ok

File: lib/service/src/main/java/io/atlasmap/service/AtlasService.java (hunk 3 above)
Comment (pleacu) [reply lib/service/src/main/java/io/atlasmap/service/AtlasService.java]: The user can import a totally different ADM file with the import icon.  Keeping ATLASMAP_ADM_PATH pointing to the original file causes it to be reread even of the user imported a different one from the UI.

File: lib/service/src/main/java/io/atlasmap/service/AtlasService.java (hunk 4 above)
Comment (pleacu) [reply lib/service/src/main/java/io/atlasmap/service/AtlasService.java]: I'll look at Files.delete()

File: lib/service/src/main/java/io/atlasmap/service/AtlasService.java (hunk 3 above)
Comment (apupier) [reply lib/service/src/main/java/io/atlasmap/service/AtlasService.java]: ok

File: lib/service/src/main/java/io/atlasmap/service/AtlasService.java (hunk 4 above)
Comment (pleacu) [reply lib/service/src/main/java/io/atlasmap/service/AtlasService.java]: decided to keep my AtlasUtil.deleteDirectoryContents() as it keeps the parent but removes all subcontent

**Pull request merged**